networkx/algorithms/link_analysis/pagerank_alg.py

```diff
@@ -45,6 +45,10 @@ def pagerank(
     tol : float, optional
       Error tolerance used to check convergence in power method solver.
 
+      Note
+      -----
+      Convergence condition is N * tol
```

Comment:
There's no indication what N is in this context.

I'd recommend copying the relevant sentence from the `Notes` section on convergence.

Reply:
got it.

Reply:
@rossbar what do you think of this?

"Convergence condition is N * tol where N is len(G). See notes for more explanation. "

Reply:
This is of course subjective, but I still don't think that's as clear as what's already in the docstring - I'd really advocating just using the existing wording!

```suggestion
      The iteration will stop after a tolerance of ``len(G) * tol`` is reached.
```

Reply:
Alright. Understood.

Reply:
@rossbar what do you think?